shap/plots/_beeswarm.py

```diff
@@ -62,23 +62,25 @@ def beeswarm(
         Axes object to draw the plot onto, otherwise uses the current Axes.
 
     show : bool
-        Whether ``matplotlib.pyplot.show()`` is called before returning.
+        Whether :external+mpl:func:`matplotlib.pyplot.show()` is called before returning.
         Setting this to ``False`` allows the plot to be customized further
-        after it has been created, returning the current axis via plt.gca().
+        after it has been created, returning the current axis via
+        :external+mpl:func:`matplotlib.pyplot.gca()`.
 
     color_bar : bool
         Whether to draw the color bar (legend).
 
     s : float
-        What size to make the markers. For further information see `s` in ``matplotlib.pyplot.scatter``.
+        What size to make the markers. For further information, see ``s`` in
+        :external+mpl:func:`matplotlib.pyplot.scatter`.
 
     plot_size : "auto" (default), float, (float, float), or None
         What size to make the plot. By default, the size is auto-scaled based on the
         number of features that are being displayed. Passing a single float will cause
         each row to be that many inches high. Passing a pair of floats will scale the
         plot by that number of inches. If ``None`` is passed, then the size of the
-        current figure will be left unchanged. If ax is not ``None``, then passing
-        plot_size will raise a Value Error.
+        current figure will be left unchanged. If ``ax`` is not ``None``, then passing
+        ``plot_size`` will raise a :external+stdlib:exc:`ValueError`.
```

Comment:
Nit: do we really need to point to the docs for simple builtin objects like these?

A slight drawback is that the raw source is a bit less readable, and I don't think it adds much value in this particular case

Reply:
Removed